packages/client/components/AnalyticsPage.tsx

```diff
@@ -80,6 +88,25 @@ if (datadogEnabled) {
 const TIME_TO_RENDER_TREE = 100
 
 const AnalyticsPage = () => {
+  const atmosphere = useAtmosphere()
+  useEffect(() => {
+    const initializeGA4 = async () => {
+      const res = await atmosphere.fetchQuery<AnalyticsPageQuery>(query)
+      if (!res) return
+      const {viewer} = res
+      const {id, segmentId} = viewer
+      ReactGA.initialize(gaMeasurementId, {
+        gaOptions: {
+          userId: id,
+          clientId: segmentId
+        }
+      })
+    }
+    if (gaMeasurementId) {
+      initializeGA4().catch()
+    }
+  })
```

Comment:
@Dschoordsch I'm not sure if this is the right place to put it, looking for suggestion! 🙏

Reply:
Looks like the correct place to me.